src/lib/admin/admin-actor-factory.ts

```diff
@@ -0,0 +1,93 @@
+import { AdminActor, AdminActorConfig } from "./admin-actor";
+import { AdminSpec } from "../data/models";
+import APP from "../app";
+import { HashService } from "../services/hash";
+import { Pool } from "pg";
+import { EncryptionService } from "../services/encryption";
+import { dbBegin, dbCommit, dbRollbackAndRelease } from "../data/dbUtils";
+import {
+  dbInsertAdmin,
+  dbSelectAdminByAdminHashedEmail,
+} from "../data/dbAdmins";
+import { AdminPii, encryptAdminPii } from "./admin-pii";
+
+const MOCK_ADMIN_ACCOUNTS: AdminPii[] = [
+  {
+    adminName: "Samantha Jones",
+    adminEmail: "admin1@admin.com",
+    adminPhoneNumber: "+65 3123 4567",
+  },
+  {
+    adminName: "Li Hui (李慧)",
+    adminEmail: "admin2@admin.com",
+    adminPhoneNumber: "+65 3234 5678",
+  },
+];
+
+async function insertMockAdminAccounts(): Promise<void> {
```

Comment:
This will be removed. A dangerous API endpoint will be implemented instead for inserting test accounts.